app/models/course/condition.rb

```diff
@@ -3,4 +3,6 @@ class Course::Condition < ActiveRecord::Base
 
   belongs_to :course, inverse_of: false
   belongs_to :conditional, polymorphic: true
+
+  delegate :satisfied_by?, to: :actable
```

Comment:
Sorry boss, I still don't get the point >.< That method in lib/extensions/conditional/active_record/base.rb:46 is to ensure that each specific condition implements `satisfied_by?`. Otherwise an error is thrown.